### DISCUSSION TITLE
풀리퀘 승인에 대한 부분

### DISCUSSION THREAD

**Discussion by Eomhyunjun** [devrel-kr/HackaLearn#284] [1 Q&A - HackaLearn 운영] [answered]
제가 sns인증을 풀리퀘를 올려놨는데 아직 승인이 안나서요.
혹시나 이게 오늘 승인이 안나면 다음 미션 3개가 밀려있는데,
풀리퀘를 1번에 1개만 보낼 수 있는데 나머지 미션 3개를 받지 못할까봐 걱정이 됩니다.
나머지 미션에 대해서 같은 풀리퀘에 커밋을 여러개 날려놔도 괜찮을까요?

**Comment by justinyoo** [comment, marked as answer]
몇 번 PR인지 알려주실 수 있으세요?

하나의 PR에는 하나의 챌린지만 올려주세요. 현재 리뷰어들이 열심히 확인 중입니다.

**Comment by Eomhyunjun** [comment]
앗 머지 됐네요 정말 감사합니다!
